client/state/data-layer/wpcom/me/notification/settings/test/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { NOTIFICATION_SETTINGS_UPDATE, NOTICE_CREATE } from 'state/action-types';
+import {
+	requestNotificationSettings,
+	updateSettings,
+	handleError,
+} from '../';
+
+describe( 'wpcom-api', () => {
```

Comment:
we could loose a couple `describes`. I've updated it on 548e60b0ec76490e32ba6bfc2ee385f644d3ff6c to have this output:

![screen shot 2017-06-09 at 14 57 14](https://user-images.githubusercontent.com/233601/26988002-06fae4ba-4d24-11e7-844f-816c082b60b7.png)

How do we feel about a test having multiple top level `describes`?